scripts/image-list.txt

```diff
@@ -11,6 +11,7 @@ docker.io/kubevirt/virt-controller:v0.35.0
 docker.io/kubevirt/virt-handler:v0.35.0
 docker.io/kubevirt/virt-launcher:v0.35.0
 docker.io/kubevirt/virt-operator:v0.35.0
+docker.io/library/busybox:1.32.0
```

Comment:
Let's remove the image-list.txt. It is expected to be generated during the build. Probably committed by mistake.